service-wcs/src/main/java/org/oskari/wcs/request/DescribeCoverage.java

```diff
@@ -0,0 +1,58 @@
+package org.oskari.wcs.request;
+
+import java.util.Map;
+import java.util.Objects;
+import java.util.Optional;
+import org.oskari.wcs.capabilities.Capabilities;
+import org.oskari.wcs.capabilities.Operation;
+import org.oskari.wcs.util.Maps;
+
+public class DescribeCoverage {
+
+    private DescribeCoverage() { /* Block */ }
+
+    /**
+     * Create query parameters for a GET encoded DescribeCoverage request You
+     * can use {@link #coverageExists(Capabilities, String)} to see if the
+     * coverage exists
+     *
+     * @param coverageId
+     *            id of the coverage
+     * @return Map with query parameters
+     */
+    public static Map<String, String> toQueryParameters(String coverageId) {
+        return Maps.of(
+                "service", "WCS",
+                "version", "2.0.1",
+                "request", "DescribeCoverage",
```

Comment:
Should DescribeCoverage, GetCapabilities etc be declared as an enum and used here (and other similar places)?